src/bootstrap/bootstrap.py

```diff
@@ -741,6 +741,9 @@ def build_bootstrap(self, color, verbose_count):
         env["LIBRARY_PATH"] = os.path.join(self.bin_root(), "lib") + \
             (os.pathsep + env["LIBRARY_PATH"]) \
             if "LIBRARY_PATH" in env else ""
+        env["LIBPATH"] = os.path.join(self.bin_root(), "lib") + \
+            (os.pathsep + env["LIBPATH"]) \
+            if "LIBPATH" in env else ""
```

Comment:
Seems OK to me - src/tools/cargo/crates/cargo-util/src/paths.rs also may want a similar change.